doc/whats_new/v0.24.rst

```diff
@@ -58,6 +58,13 @@ Changelog
   sparse matrix or dataframe at the start. :pr:`17546` by
   :user:`Lucy Liu <lucyleeow>`.
 
+- |Enhancement| Add `ensemble` parameter to
+  :class:`calibration.CalibratedClassifierCV`, which enables implementation
+  of calibration via an ensemble of calibrators (current method) or
+  just one calibrator using all the data (similar to the built-in feature of
+  :mod:`sklearn.svm` estimators with the `probabilities=True` parameter).
+  :pr:`17856` by :user:`Lucy Liu <lucyleeow>`.
```

Comment:
I think we should also credit @aesuli for the original issue and PR

(sorry, I don't have time to make another real review at the moment)

Reply:
Thanks I forgot!